packages/integrations/vercel/src/static/adapter.ts

```diff
@@ -22,6 +22,10 @@ export default function vercelStatic(): AstroIntegration {
 			'astro:config:done': ({ setAdapter, config }) => {
 				setAdapter(getAdapter());
 				_config = config;
+
+				if(config.output === 'server') {
+					throw new Error(`${PACKAGE_NAME} should be used with output: 'static'`);
```

Comment:
Can you explain this? Does this mean there is Vercel adapter needed for static build?

I am experimenting with Vercel without adapters and Im having trouble with "command astro not found issue" which is in my dependencies.

Is this adapter meant to fix issues like this?